libs/designer/src/lib/core/DesignerProvider.tsx

```diff
@@ -22,15 +22,7 @@ const OptionsStateSet = ({ options, children }: any) => {
   const dispatch = useDispatch();
   useEffect(() => {
     if (!options) return;
-    dispatch(
-      initDesignerOptions({
-        readOnly: options.readOnly,
-        isMonitoringView: options.isMonitoringView,
-        isDarkMode: options.isDarkMode,
-        useLegacyWorkflowParameters: options.useLegacyWorkflowParameters,
-        isXrmConnectionReferenceMode: options.isXrmConnectionReferenceMode,
-      })
-    );
+    dispatch(initDesignerOptions(options));
```

Comment:
Do we need to update the designerOptionsSlice for the suppressDefaultNodeSelectFunctionality and addedNodeSelectCallback?

Reply:
Switching it over to this implementation makes it so we don't have to specify each option anymore

Reply:
Yep, thats great, but I wonder if on the execution of `initDesignerOptions` we need to specify this new properties.
<img width="1120" alt="image" src="https://github.com/Azure/LogicAppsUX/assets/102700317/796adfbb-b616-4913-a213-d3d73004d0d6">